packages/router/src/provide_router.ts

```diff
@@ -269,8 +269,9 @@ const BOOTSTRAP_DONE = new InjectionToken<Subject<void>>(
  * configured through the `ExtraOptions`.
  *
  * When set to `EnabledBlocking`, the initial navigation starts before the root
- * component is created. The bootstrap is blocked until the initial navigation is complete. This
- * value is required for [server-side rendering](guide/ssr) to work.
+ * component is created. The bootstrap is blocked until the initial navigation is complete. If not
+ * using [hydration](guide/hydration), this value is required for [server-side rendering](guide/ssr)
+ * to work.
```

Comment:
I'd like to propose rephrasing it a bit (and similarly in other locations that you've updated in this PR):
```suggestion
 * component is created. The bootstrap is blocked until the initial navigation is complete. This value
 * should be set in case you use [server-side rendering](guide/ssr), but do not enable
 * [hydration](guide/hydration) for your application.
```

Reply:
Yup, much clearer indeed. done in 7d7bb08353e02bf26640f502e86a14e08dea4fc8

In the provider docs I omitted `value` as it's a function that sets the value for the injection token